apps/demos/Demos/DataGrid/SimpleArray/Angular/app/app.component.ts

```diff
@@ -16,8 +16,12 @@ if (!/localhost/.test(document.location.host)) {
 export class AppComponent {
   customers: Customer[];
 
+	columns: string[];
+
   constructor(service: Service) {
     this.customers = service.getCustomers();
+		
+		this.columns = ['CompanyName', 'City', 'State', 'Phone', 'Fax'];
```

Comment:
Let's change the indent, it looks too big + remove the empty line:
```suggestion
    this.columns = ['CompanyName', 'City', 'State', 'Phone', 'Fax'];
```